src/underactuated/pyplot_visualizer.py

```diff
@@ -42,8 +43,8 @@ def __init__(self, draw_rate=0.0333, facecolor=[1, 1, 1]):
     def _DoPublish(self, context, event):
         self.draw(context)
         self.fig.canvas.draw()
-        if (plt.get_backend() == u'MacOSX'):
-            plt.pause(1e-10)  # Needed to see anything on mac.
+        if plt.get_backend() != u'template':
```

Comment:
is that important?  can we just do it always?  presumably the template would just ignore it?

Reply:
The template backend throws a not-implemented error if I try to call that -- happened during CI before I added this

Reply:
i figured, but wanted to understand.  thanks!